base/loading.jl

```diff
@@ -508,6 +508,8 @@ package root.
 To get the root directory of the package that implements the current module
 the form `pkgdir(@__MODULE__)` can be used.
 
+If an extension module is given, the root of the parent package is returned.
```

Comment:
Why shouldn't it give the path to the extension?

Reply:
Extensions aren't fully fledged packages themselves, right? The resulting path would not point to a package.

Reply:
Well, they load like a normal package at least. But I see that `pkgdir` gives you the path to where the whole package is stored so it would be hard to have an analogy for that for an extension so maybe this is the best thing to do.